src/tribler/gui/network/request/file_download_request.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+from PyQt5.QtNetwork import QNetworkRequest
+
+from tribler.gui.network.request.request import Request
+
+
+class FileDownloadRequest(Request):
```

Comment:
I think `FileDownloadRequest` is unnecessary, as it is possible to do the same with a base `Request` class